client/state/automated-transfer/constants.js

```diff
@@ -1,7 +1,16 @@
 export const AUTOMATED_TRANSFER_STATUS = {
+	INQUIRING: 'inquiring',
 	START: 'start',
 	SETUP: 'setup',
 	LEAVING: 'leaving',
 	CONFLICTS: 'conflicts',
 	COMPLETE: 'complete',
 };
+
+export const eligibilityHolds = {
+	NO_WPCOM_NAMESERVERS: 'NO_WPCOM_NAMESERVERS',
+	NON_ADMIN_USER: 'NON_ADMIN_USER',
+	NOT_USING_CUSTOM_DOMAIN: 'NOT_USING_CUSTOM_DOMAIN',
+	SITE_GRAYLISTED: 'SITE_GRAYLISTED',
+	SITE_PRIVATE: 'SITE_PRIVATE',
+};
```

Comment:
More a question about Calypso conventions re: objects of constants.

You used:
- name in camel case
- properties in upper snake
- values in upper snake

I used (for `AUTOMATED_TRANSFER_STATUS`):
- name in upper snake
- properties in upper snake
- values in lower (likely kebab, as those values will be used in a component attributes)

Is there a rule to follow?
I could modify `AT_STATUS` if needed.